### DISCUSSION TITLE
Allow using a separate cover storage folder

### DISCUSSION THREAD

**Discussion by p3732** [jcorporation/myMPD#213] [Q&A] [answered]
I recently found this great project and I would really like to use the cover grid view for my setup, but I am facing the problem that mympd does not have write access to the library, since it is mounted via nfs.

Now I clearly am not very knowledgable about how mympd stores the cover information, but what I would find very useful is to be able to define a directory (via mympd.conf) where the covers should be placed. I think this might in general be a good feature for people that want to keep their music library untouched by their music player.

Very curious about your opinion on this, thank you!

**Comment by jcorporation** [comment, marked as answer]
myMPD don't write anything to the music directory and supports only local albumart, it downloads no covers from the internet. You have to place the albumart in the music directory yourself. myMPD supports embedded albumart or pictures in the album folders. 

For details: https://github.com/jcorporation/myMPD/wiki/Albumart

I am using https://github.com/sahib/glyr for downloading albumart.

**Comment by p3732** [comment]
That is great, thank you for the explanation! I did not see that there is a Wiki, so I created an MR to add it more publicly. Also thanks for the glyr hint!
